app/helpers/patient_details_helper.rb

```diff
@@ -65,6 +65,7 @@ def linelist_for_export
       user_defined_id_statelocal: user_defined_id_statelocal || '',
       sex: sex || '',
       date_of_birth: date_of_birth&.strftime('%F') || '',
+      primary_telephone: primary_telephone || '',
```

Comment:
can you move `primary_telephone` to the end if it's going to be the last column in the export

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1164/commits/735d0d794dccc9773fcc41cddffc0619a4fa5e03